resources/docs/livewire/creating-chirps.md

```diff
@@ -0,0 +1,504 @@
+
+To allow users to post Chirps, we will need to create models, migrations, and controllers. Let's explore each of these concepts a little deeper:
+
+* [Models](https://laravel.com/docs/eloquent) provide a powerful and enjoyable interface for you to interact with the tables in your database.
+* [Migrations](https://laravel.com/docs/migrations) allow you to easily create and modify the tables in your database. They ensure that the same database structure exists everywhere that your application runs.
+* [Controllers](https://laravel.com/docs/controllers) are responsible for processing requests made to your application and returning a response.
+
+Almost every feature you build will involve all of these pieces working together in harmony, so the `artisan make:model` command can create them all for you at once.
+
+Let's create a model, migration, and controller for our Chirps with the following command:
+
+```shell
+php artisan make:model -mc Chirp
+```
+
+> **Note**
+> You can see all the available options by running the `php artisan make:model --help` command.
+
+This command will create three files for you:
+
+* `app/Models/Chirp.php` - The Eloquent model.
+* `database/migrations/<timestamp>_create_chirps_table.php` - The database migration that will create your database table.
+* `app/Http/Controller/ChirpController.php` - The HTTP controller that will take incoming requests and return responses.
+
+## Routing
+
+
+We will also need to create URLs for our controller. We can do this by adding "routes", which are managed in the `routes` directory of your project.
+
+Now, because we're using Livewire, we only need to enable a single `Route::get` route to display our form and a listing of Chirps. Additionally, we are going to place this route behind two [middleware](https://laravel.com/docs/middleware):
+
+* The `auth` middleware ensures that only logged-in users can access the route.
+* The `verified` middleware will be used if you decide to enable [email verification](https://laravel.com/docs/verification).
+
+```php filename=routes/web.php
+<?php
+
+use App\Http\Controllers\ChirpController;// [tl! add]
+use Illuminate\Support\Facades\Route;
+// [tl! collapse:start]
+/*
+|--------------------------------------------------------------------------
+| Web Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register web routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| contains the "web" middleware group. Now create something great!
+|
+*/
+
+Route::get('/', function () {
+    return view('welcome');
+});
+// [tl! collapse:end]
+Route::get('/dashboard', function () {
+    return view('dashboard');
+})->middleware(['auth', 'verified'])->name('dashboard');
+// [tl! add:start]
+Route::get('/chirps', [ChirpController::class, 'index'])
+    ->middleware(['auth', 'verified'])
+    ->name('chirps'); // [tl! add:end]
```

Comment:
Should we use `Volt::route` method and skip the controller? It's nice to expose them to controllers, but it feels like extra steps you probably wouldn't do if building this for real. They can always learn about controllers in the other stacks.

Reply:
I've considered that. However, still feel it's important to mention controllers on this tutorial as the real entry point of your application.